config/initializers/geocoder.rb

```diff
@@ -1,6 +1,6 @@
 Geocoder.configure(
-  lookup: :uk_ordnance_survey_names,
-  api_key: ENV.fetch("ORDNANCE_SURVEY_API_KEY", nil),
+  lookup: :google,
+  api_key: ENV.fetch("GOOGLE_LOCATION_SEARCH_API_KEY", ""),
```

Comment:
am assuming its ok that second argument is empty string here but was nil before?

Reply:
If the given name does not exist and neither default nor a block a provided an KeyError is raised.

> I think this is just to prevent an error if the env var isn't found? I'm not too sure it matters if it's an empty string or nil.

In google_api.rb empty strings are used so I just copied that.

```
GOOGLE_API_JSON_KEY = ENV.fetch("GOOGLE_API_JSON_KEY", "")
GOOGLE_ANALYTICS_PROFILE_ID = ENV.fetch("GOOGLE_ANALYTICS_PROFILE_ID", "")
GOOGLE_LOCATION_SEARCH_API_KEY = ENV.fetch("GOOGLE_LOCATION_SEARCH_API_KEY", "")
```

Reply:
im sure its fine, just noticed thats all